aten/src/ATen/Context.h

```diff
@@ -387,4 +387,14 @@ struct TORCH_API NoTF32Guard {
   bool changed = false;
 };
 
+#ifdef USE_ROCM
+struct TORCH_API BackwardPassGuard {
```

Comment:
Would it be acceptable to keep it named `BackwardPassGuard` without ROCm in the name?  We have downstream pytorch extensions that use this feature already, and it would have a ripple effect.  Nothing that couldn't be resolved, of course.  But it would minimize our developer effort to keep the name as-is.  Let me know.  Thanks.

Reply:
Ho that's annoying... If these projects use custom build they will have to be updated to master anyways?

Reply:
Renamed `BackwardPassGuard` to `ROCmBackwardPassGuard` in https://github.com/pytorch/pytorch/pull/71881/commits/e4e2c0e8062a29019fe10395b7fca8d8f176145b.